### DISCUSSION TITLE
Authentication with data from a customer collection

### DISCUSSION THREAD

**Discussion by cybertaz** [directus/directus#7674] [Archive]
Hello, I'm new to this and I'm evaluating Directus for our e-commerce system (website, admin, mobile apps, etc). 

We have a mysql db with records of over 20 years, therefore I need to use directus on top of it, so I need to authenticate 2 types of people Admin Users (order processing, inventory, etc) and Customers (place orders, view previous orders, update their profile, etc). I would like to use directus just as an api to my existing database. All the front ends will be built as separate apps.

I would like to authenticate the Customers using the customers collection instead of the users collection... and use the users collection for the admin users.

I saw that a use case for e-commerce on your website, but there are no details on how to use directus from that perspective. Do you have any documentation on how to go about achieving this?

At the moment I want to build a new website and keep our existing admin system but both working with the same database (new website through directus api and the admin system directly with the mysql db).

Please guide me to get this done.

Thank you for your help, is greatly appreciate it.

**Comment by rijkvanzanten** [comment]
In order to rely on Directus' permissions and authentication, both those user types will have to live under `directus_users`. Permissions are controlled through Roles in which those users reside. It's currently not possible to use (different) user-collections for different roles / user sets

**Reply by cybertaz** [reply]
Ok. Thank you.
